### DISCUSSION TITLE
[Help] Downloads button on main taskbar not disappearing? Requires restart

### DISCUSSION THREAD

**Discussion by anEleadE** [Eloston/ungoogled-chromium#2702] [Q&A] [answered]
Hi all,

![Screenshot 2024-02-05 at 19 29 09](https://github.com/ungoogled-software/ungoogled-chromium/assets/107296855/bf6917f3-7cd7-4838-842a-887d467da05d)

As in my screenshot, the Downloads button on Ungoogled-Chromium is not able to be dismissed. If I clear all the downloads, it stays on the bar, regardless of how long has passsed.

The only way to remove it is to force quit Ungoogled-Chromium and relaunch. 

**Version** 121.0.6167.85 (Official Build, ungoogled-chromium) (arm64)
**Machine** Macbook Air M1
**OS** MacOS Sonoma Version 14.2.1 (23C71)

Can anyone share any advice that might help?

Thank you!

**Comment by PF4Public** [comment]
Have you  tried the same in Chromium? Most likely it is Chromium behaviour.

**Comment by anEleadE** [comment, marked as answer]
> Have you tried the same in Chromium? Most likely it is Chromium behaviour.

You were right. The only way to get the downloads button to leave is to head to chrome://downloads and clear everything out there. 

Thanks, resolved.
